aloe_webdriver/util.py

```diff
@@ -255,6 +255,12 @@ def field_xpath(field, attribute):
         xpath = './/{field}[@{attr}=%s]'
 
     else:
+        # The string 'input-button' is used to distinguish a search
```

Comment:
Thanks for two good ideas. I've updated the pull request with both of these implemented.